src/pheval/cli_pheval_utils.py

```diff
@@ -27,3 +34,77 @@ def scramble_semsim():
 def scramble_phenopacket():
     """scramble_phenopacket"""
     print("running pheval_utils::scramble_phenopacket command")
+
+
+@click.command()
+@click.option(
+    "--left",
+    "-l",
+    required=True,
+    metavar="FILE",
+    help="Path of first file that will be compared",
+)
+@click.option(
+    "--right",
+    "-r",
+    required=True,
+    metavar="FILE",
+    help="Path of second file that will be compared",
+)
+@click.option(
+    "--join-columns",
```

Comment:
Do not use format-specific terminology like "column". Most likely, the inputs will be tables in _this case_ but its better to write generic CLIs so we can, at a later stage, include semantic similarity profiles in other formats, like JSON.